Orange/widgets/unsupervised/owmanifoldlearning.py

```diff
@@ -93,11 +93,29 @@ class TSNEParametersEditor(ManifoldParametersEditor):
     # rename l2 to Euclidean
     metric_values = [("l2", "Euclidean")] + metric_values
 
+    perplexity = Setting(30)
+    early_exaggeration = Setting(4)
+    learning_rate = Setting(1000)
+    n_iter = Setting(1000)
```

Comment:
I suggest we have the same defaults as the latest sklearn version (they have been changing them recently)
perplexity=30, e.e.=12, lr=200, iter=1000

Reply:
I didn't realize they had changed them. Fixed now.